### ISSUE TITLE
NTR: 2-methyl-3-hydroxybutyric acid high in urine (Lab: chemistry: organic acids)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8588]
New term request 2-methyl-3-hydroxybutyric acid high in urine (segal_030509192906): Acetoacyl-CoA thiolase

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000603
- label: Elevated urinary 2-methyl-3-hydroxybutyric acid level
- parents: [Increased urine proteinogenic amino acid derivative level (HP:0033097)](https://hpo.jax.org/app/browse/term/HP:0033097)
- pmid: PMID:9700610
- def: The amount of 2-methyl-3-hydroxybutyric acid in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 2-methyl-3-hydroxybutyric acid is a normal urinary metabolite involved in the isoleucine catabolism and is excreted in abnormally high amounts in alpha-methylacetoacetic aciduria.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8588
- synonyms:
